configs/templates/ufs-srw-public-v2/spack.yaml

```diff
@@ -20,23 +20,24 @@ spack:
   specs:
```

Comment:
The changes to this file are unrelated to the containers. They come from the original Intel container branch from @mark-a-potts . This file isn't used in CI testing, there is `ufs-srw-dev` instead.